npm2deb/__init__.py

```diff
@@ -434,7 +448,8 @@ def download(self):
         tarball.close()
 
         # remove tarball file
-        _os.remove(tarball_file)
+        if not self.noclean
```

Comment:
Again, `:` is missing

Reply:
If you didn't already pick up on this, I am not a Python native.  Sorry about that.